modules/mobile/app/controllers/mobile/v0/check_in_controller.rb

```diff
@@ -12,7 +12,7 @@ def create
         if (error = parse_error(attributes))
           raise(*error)
         else
-          render json: Mobile::V0::CheckInSerializer.new(@current_user.id, attributes)
```

Comment:
This was one of the issues that prompted the SIS spec switch. This method only exists on IAM users.